recipes/repeatmodeler/run_test.sh

```diff
@@ -2,10 +2,13 @@
 set -x -e
 
 # build the db with BuildDatabase
-BuildDatabase -engine rmblast -name db test.fa
+# BuildDatabase -engine rmblast -name db test.fa
```

Comment:
Why were these tests disabled? It seems like this would be something useful to check it works as expected.  (likewise for the repeatmodeller test below)

Reply:
While testing locally, RepeatModeler runs into an error pointing to a missing 'setThreadByQuery' module in RepeatMasker's NCBIBlastSearchEngine.pm, which was introduced in RepeatMasker v4.1.4 and to accomodate multi-threading in RMBlast 2.13.0. The current recipe for RepeatMasker is v4.1.2p1 and these tests were disabled to workaround the missing setThreadByQuery module.
Fortunately, it looks like #40484 is working on updating RepeatMasker and should resolve the missing module.

Reply:
Fair enough, a comment in the code to that effect would be good, or keep the tests and wait on merging this till the linked repeatmasker PR is merged